src/lib/replication/service/generic_repl_svc.cpp

```diff
@@ -36,15 +37,20 @@ std::shared_ptr< GenericReplService > GenericReplService::create(cshared< ReplAp
 
 GenericReplService::GenericReplService(cshared< ReplApplication >& repl_app) :
         m_repl_app{repl_app}, m_my_uuid{repl_app->get_my_repl_id()} {
+    m_sb_bufs.reserve(100);
```

Comment:
some dynamic config MAX_NUM_PG_IN_SYSTEM ?

Reply:
This is just for reserve and not for any tunables? Do we need to clutter the dynamic config with this (also homestore doesn't know anything about PG).